src/modules/core/components/BannedTag/BannedTag.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { HTMLAttributes } from 'react';
+import { FormattedMessage, MessageDescriptor } from 'react-intl';
+
+import Icon from '~core/Icon';
+
+import styles from './BannedTag.css';
+
+const displayName = 'BannedTag';
+
+interface Props extends HTMLAttributes<HTMLSpanElement> {
+  text?: MessageDescriptor | string;
+  textValues?: { [key: string]: string };
+}
+
+const BannedTag = ({ children, text, textValues, ...rest }: Props) => {
```

Comment:
Please add a default value to the `text` prop, the label I've told you about.

```suggestion
const BannedTag = ({
  children,
  text = { id: 'label.banned' },
  textValues,
  ...rest
}: Props) => {
```

Reply:
Addressed in latest commit